### PR TITLE
Improve Microsoft.Insights schema generation

### PR DESCRIPTION
**Pull request by d3r3kk**
**Note**: No new or updated API code, just config changes to improve ARM schema generation for Application Insights.

Changes:
- Case of microsoft.insights updated to Microsoft.Insights throughout.
- Added schema tag to applicationinsights/resource-manager/readme.md.
  - Limits the input files read from during schema generation.
  - Limited to only those APIs that comply with current ARM schema rules.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
    - not applicable here.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -230,17 +230,17 @@ These settings apply only when `--tag=package-2015-05` is specified on the comma
...
-- microsoft.insights/stable/2015-05-01/componentApiKeys_API.json
-- microsoft.insights/stable/2015-05-01/componentContinuousExport_API.json
-- microsoft.insights/stable/2015-05-01/componentFeaturesAndPricing_API.json
-- microsoft.insights/stable/2015-05-01/componentProactiveDetection_API.json
-- microsoft.insights/stable/2015-05-01/components_API.json
-- microsoft.insights/stable/2015-05-01/componentWorkItemConfigs_API.json
-- microsoft.insights/stable/2015-05-01/favorites_API.json
-- microsoft.insights/stable/2015-05-01/webTestLocations_API.json
-- microsoft.insights/stable/2015-05-01/webTests_API.json
+- Microsoft.Insights/stable/2015-05-01/aiOperations_API.json
```
Comment (lmazuel) [inline specification/applicationinsights/resource-manager/readme.md]: These files can be read from Windows or Ubuntu, meaning file name is case-sensitive. This is why the CI is failing.

Comment (d3r3kk) [reply specification/applicationinsights/resource-manager/readme.md]: Yes, I see. Changing the case of the folder is my intention here. Any tips on getting folder `microsoft.insights` changed to `Microsoft.Insights`?

Comment (dsgouda) [reply specification/applicationinsights/resource-manager/readme.md]: @d3r3kk  You might want to give [this](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) a try

Comment (d3r3kk) [reply specification/applicationinsights/resource-manager/readme.md]: @dsgouda Thank you!!! 🥇 I appreciate the help.

**Comment by d3r3kk** [comment]
@lmazuel, any further feedback?

**Comment by lmazuel** [comment]
@d3r3kk I'm not the reviewer, I trust @dsgouda 100 %. I was just around to discover why the Python job failed :). Now Python job is ok, and I'm happy ;)

**Comment by dsgouda** [comment]
Taking a look now

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**